src/match/models.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth.models import User
+from django.db import models
+from location.models import Location
+from match.validators import validate_int_list
+from match.validators import validate_times_list
+
+
+class Match(models.Model):
+    status = models.CharField(max_length=20)
+    user_1 = models.ForeignKey(
+        User, default=None, null=True, on_delete=models.CASCADE, related_name="+"
+    )
+    user_2 = models.ForeignKey(
+        User, default=None, null=True, on_delete=models.CASCADE, related_name="+"
```

Comment:
related name shouldn't just be a `+`

the related name is like if you found a `User` object and tried to access the `match` 
notice how i can access `User` through `match` with `user_2`
so then the related object must be of `match` type right